package-lock.json

```diff
@@ -5065,25 +5065,25 @@
           "dependencies": {
             "abbrev": {
               "version": "1.1.1",
-              "resolved": false,
+              "resolved": "https://registry.npmjs.org/abbrev/-/abbrev-1.1.1.tgz",
```

Comment:
The diff in this file looks unintentional, try running `npm install` and it should go away.

Reply:
Actually I think I did run `npm install`. I think this didn't happen during the last dependency update, but is now covered by Isaac's currently active PR.